styles/github/catppuccin.user.css

```diff
@@ -501,6 +501,23 @@
     --shadow-floating-legacy: 0px 6px 12px -3px @crust, 0px 6px 18px 0px @crust;
     --outline-focus: @blue solid 2px;
 
+    /* Customer Agreements */
+    .enterprise-customer-agreement {
+      color: @accent-color !important;
+
+      &:hover {
+        .octicon {
+          color: @accent-color !important;
+        }
+      }
+
+      h3 {
+        &:hover {
+          text-decoration: underline !important;
+        }
+      }
```

Comment:
We don't need to set this.

Reply:
If we do not set it, the underline does not appear on hover, and thus it will not match the style of other linked headings.

Reply:
Userstyles policy is to not change any non-color related settings. We simply go wtih what they have for the link styling existing on that page.

Reply:
But now that that is applied, can you merge? I can't request another review.